Dockerfile.devel

```diff
@@ -0,0 +1,25 @@
+
+FROM dmattli/debian-cuda:10.0-buster-runtime AS runtime
```

Comment:
Good idea to separate devel from runtime! Although this line will need to be updated once, I rebuild the upstream image.